uk.co.compucorp.civicrm.hrleaveandabsences/templates/CRM/HRLeaveAndAbsences/Page/AbsenceTab.tpl

```diff
@@ -1,10 +1,53 @@
 <section data-leave-absences-absence-tab>
-  <ui-view></ui-view>
+  <div class="chr_absence-tab-page">
+    <div class="button-container">
+      <div class="btn-group">
+        <button type="button" class="btn btn-primary">Create New</button>
+        <button
+          type="button"
+          class="btn btn-primary dropdown-toggle dropdown-toggle-split"
+          data-toggle="dropdown">
+          <span class="caret"></span>
+        </button>
+        <ul class="dropdown-menu">
+          <li>
+            <a href="#">Record staff sickness</a>
+          </li>
+          <li>
+            <a href="#">Record staff overtime</a>
+          </li>
+        </ul>
+      </div>
+    </div>
+
+    <div class="tabs-container">
+      <ul class="nav nav-tabs nav-tabs-header">
+        <li>
+          <a class="chr_absence-tab-page--active_bottom">Report</a>
```

Comment:
`chr_absence-tab-page--active_bottom` should not be created. Active class is already present in tabs, see how it is done for My Leave and Manager leave(Blue bottom border). And just override it.